strawberry/tools/create_type.py

```diff
@@ -1,11 +1,19 @@
 import types
-from typing import List, Type
+from typing import List, Optional, Sequence, Type
 
 import strawberry
 from strawberry.field import StrawberryField
 
 
-def create_type(name: str, fields: List[StrawberryField]) -> Type:
+def create_type(
+    name: str,
+    fields: List[StrawberryField],
+    is_input: bool = False,
+    is_interface: bool = False,
+    description: Optional[str] = None,
+    directives: Optional[Sequence[object]] = (),
+    extend: bool = False,
+) -> Type:
```

Comment:
Maybe return `Type[WithStrawberryObjectDefinition]` in here since we have that now?